public/app/features/dashboard/custom_time_ranges/TimeRangeList.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { PureComponent } from 'react';
+
+const width1 = {
+  width: '1%',
+};
+
+const lightBlue = {
+  color: '#32d1df',
+};
+
+const green = {
+  color: '#269242',
+};
+
+export interface Props {
+  ranges: any;
+  up: any;
+  down: any;
+  edit: any;
+  delete: any;
+  add: any;
+}
+
+export default class List extends PureComponent<Props> {
+  constructor(props) {
+    super(props);
+    this.state = { show: false };
+  }
+
+  clickUp(index) {
+    this.props.up(index);
+  }
+  clickDown(index) {
+    this.props.down(index);
+  }
+  clickEdit(index) {
+    this.props.edit(index);
+  }
+  clickDelete(index) {
+    this.props.delete(index);
+  }
+  setupNew = () => {
+    this.props.add();
+  };
+
+  render() {
+    const NewDay = props => {
+      return (
+        <span style={green} className="pointer">
+          Starts a new day
+        </span>
+      );
+    };
+
+    const rowLen = this.props.ranges.length;
+    const tableRows = this.props.ranges.map((range, index) => {
+      return (
+        <tr key={index}>
+          <td className="pointer" onClick={() => this.clickEdit(index)}>
+            <i className="fa fa-clock-o" />
+            <span>&nbsp;&nbsp;shift</span>
+          </td>
+          <td onClick={() => this.clickEdit(index)}>
+            <span className="pointer">
+              <strong style={lightBlue}>Name: </strong>
+              {range.name}
+              <span>&nbsp;&nbsp;</span>
+            </span>
+            <span className="pointer">
+              <strong style={lightBlue}>From: </strong>
+              {range.from}
+              <span>&nbsp;&nbsp;</span>
+            </span>
+            <span className="pointer">
+              <strong style={lightBlue}>To: </strong>
+              {range.to}
+              <span>&nbsp;&nbsp;</span>
+            </span>
+            {range.newDay ? <NewDay /> : null}
```

Comment:
instead of a ternary operator you can use the `&&` operator. 

`range.newDay ? <NewDay /> : null` -> `range.newDay && (<NewDay />)`